conf/char_athena.conf

```diff
@@ -144,6 +144,10 @@ unknown_char_name: Unknown
 // To log the character server?
 log_char: yes
 
+// Minimum length for a character name.
+// Must be set to '4' unless your client uses the 'Remove 4/6 letter Character Name limit' diff patch.
+char_name_min_length: 4
```

Comment:
Is that one not packetver dependent, like the rest?

Reply:
Despite the name of the diff, no, it's still 4 letters in the new UI